lib/class-wp-rest-widgets-controller.php

```diff
@@ -769,44 +672,64 @@ public function get_item_schema() {
 					'required'    => true,
 					'context'     => array( 'view', 'edit', 'embed' ),
 				),
+				'rendered'      => array(
+					'description' => __( 'HTML representation of the widget.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'readonly'    => true,
+				),
+				'rendered_form' => array(
+					'description' => __( 'HTML representation of the widget admin form.', 'gutenberg' ),
+					'type'        => 'string',
+					'context'     => array( 'edit' ),
+					'readonly'    => true,
+				),
+				'instance'      => array(
+					'description' => __( 'Instance settings of the widget, if supported.', 'gutenberg' ),
+					'type'        => 'object',
+					'context'     => array( 'view', 'edit', 'embed' ),
+					'default'     => null,
+				),
+				'form_data'     => array(
+					'description' => __( 'Serialized form data from the widget admin form. Used to update a widget that does not support instance. Write only.', 'gutenberg' ),
```

Comment:
I'd specify the type of serialization in the description.

Reply:
👍 a0d7eb5b0e3b6fa876623e4e030e7160e00c1cfe